### PR TITLE
Bug 1207526 - [browser] add pocket shortcut in app deck, r=rexboy

### PR DESCRIPTION
**Pull request by danhuang1202**
- add pocket application
- update openPage() in browser/js/connection_handler.js

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80429d2d01e8a3106a9570023e00c617efaf1dd4

File: tv_apps/pocket/js/pocket.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    });
+
+    // XXX: smart-system application opening transition would failed,
+    // if the application exit quickly. Bug 1208414 would be use to
+    // trace this issue.
+    setTimeout(function(){
+      self.close();
+    }, 1500);
+  });
+};
```
Comment (rexboy7) [inline tv_apps/pocket/js/pocket.js]: nit: missing newline

File: tv_apps/pocket/manifest.webapp [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "en-US": {
+      "name": "Pocket",
+      "description": "Pocket"
+    }
+  },
+  "default_locale": "en-US",
+  "icons": {
+    "336": "/style/icons/pocket_336.png"
+  }
+}
```
Comment (rexboy7) [inline tv_apps/pocket/manifest.webapp]: nit: missing newline.

File: tv_apps/pocket/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+  <meta charset="utf-8">
+  <title>pocket</title>
+  <script defer src="js/pocket.js"></script>
+</head>
+<body></body>
+</html>
```
Comment (rexboy7) [inline tv_apps/pocket/index.html]: nit: missing newline.

File: tv_apps/pocket/manifest.webapp [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Pocket",
+  "description": "Pocket",
+  "launch_path": "/index.html",
+  "type": "certified",
+  "permissions": {
+    "browser":{},
```
Comment (rexboy7) [inline tv_apps/pocket/manifest.webapp]: Is this permission required?

File: tv_apps/pocket/manifest.webapp [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "systemXHR":{},
+    "desktop-notification" : {}
```
Comment (rexboy7) [inline tv_apps/pocket/manifest.webapp]: Actually I think you can remove the permissions section completely.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a69819d26e8d01a6a9349707115eb737ca4aec56

**Comment by danhuang1202** [comment]
I had updated all nits in this patch.

**Pull request merged**